app/models/api_key.rb

```diff
@@ -10,7 +10,7 @@ class ApiKey < ApplicationRecord
   validates :value, uniqueness: true
   validates :value, presence: true, on: :update
 
-  scope :active, -> { where('expires_at IS NULL OR expires_at > ?', Time.current) }
```

Comment:
I think you can keep the scope and maybe also add a `expired` scope? 🤔 and then just use .active in the default_scope? (I prefer not thinking and having the name `active` is really easy to understand.)

Reply:
Don't mind keeping `.active` but I'd prefer to delay adding `.expired` until it will be needed in some feature.